build.assets/windows/build.ps1

```diff
@@ -444,6 +444,44 @@ function Build-Connect {
     Write-Host $("Built Teleport Connect in {0:g}" -f $CommandDuration)
 }
 
+function Write-Version-Objects {
+    [CmdlesBinding()]
+    param(
+        [Parameter(Mandatory)]
+        [string] $TeleportSourceDirectory,
+        [Parameter(Mandatory)]
+        [string] $TeleportVersion,
+    )
+    Write-Host "Generating version info files for tsh.exe and tctl.exe..."
+
+    # install go-winres (v0.3.3)
+    go install github.com/tc-hib/go-winres@d743268d7ea168077ddd443c4240562d4f5e8c3e
+
+    $Year = (Get-Date).Year
+
+    # generate tsh version info
+    go-winres simply --no-suffix --arch amd64 `
```

Comment:
I'm not seeing any of these args/flags in the GH readme for this tool. Where are they coming from?

Reply:
```
➜ go-winres simply -h
NAME:
   go-winres simply - Make syso files for the "go build" command (simplified)

USAGE:
   go-winres simply [command options]

OPTIONS:
   --arch value               comma separated list of target architectures (default: "amd64,386") [$GOARCH]
   --out value                name or prefix of the object file (syso) (default: "rsrc")
   --no-suffix                don't add target suffixes such as "_windows_386" (default: false)
   --product-version value    set product version (special value: "git-tag")
   --file-version value       set file version (special value: "git-tag")
   --manifest value           type of manifest: "cli", "gui" or "none" (default: "cli")
   --admin                    set execution level to "Require Administrator" (default: false)
   --file-description value   set file description
   --product-name value       set product name
   --copyright value          set copyright notice
   --original-filename value  original filename
   --icon value               icon file (ico, png, ...) (default: "winres/icon.png")
   --help, -h                 show help
```